lang/contexts.json

```diff
@@ -4,11 +4,13 @@
 	"Insert column left": "Label for the insert table column to the left of the current one button.",
 	"Insert column right": "Label for the insert table column to the right of the current one button.",
 	"Delete column": "Label for the delete table column button.",
+	"Select entire column": "Label for the select table entire column button.",
```

Comment:
@Reinmar yay/no

Reply:
Yes, I think it can be "select column".

In fact, it should be "select column" or "select columns" depending on the selection, but the same applies to the other labels (e.g. delete column) and we ignored this for the moment.